tests/integration/remote/invoke/test_remote_invoke.py

```diff
@@ -396,11 +392,12 @@ def test_invoke_empty_event_provided(self, resource_id, expected_response):
     @parameterized.expand(
         [
             ("ChildStack/HelloWorldFunction", '{"key1": "Hello", "key2": "world"}', {"message": "Hello world"}),
-            # Skip remote invoke Step function integration tests
-            # ("ChildStack/HelloWorldStateMachine", '{"key1": "Hello", "key2": "world"}', "World"),
+            ("ChildStack/HelloWorldStateMachine", '{"key1": "Hello", "key2": "world"}', "World"),
         ]
     )
     def test_invoke_with_event_provided(self, resource_id, event, expected_response):
+        if self.stack_resource_summaries[resource_id].resource_type not in self.supported_resources:
+            pytest.skip()
```

Comment:
Thanks for addressing this, can we also add message into `skip` call why this is been skipped. If you can't find it with `pytest.skip` there should be also one with `self.skip` (or something like that).

Reply:
Yes can do  :+1: